lib/generators/blacklight/solr4_generator.rb

```diff
@@ -10,10 +13,8 @@ class Solr4Generator < Rails::Generators::Base
     EOS
 
     def install_jettywrapper
-      return unless options[:jettywrapper]
```

Comment:
Not sure if this is controversial at all, but in the context that I could find, this is already checked before this generator is run. (See https://github.com/projectblacklight/blacklight/blob/master/lib/generators/blacklight/install_generator.rb#L30)

Not sure if there are other contexts where this would be run but we would not want to install jettywrapper (and only run this generator to install rsolr).